stdlib/LinearAlgebra/test/qr.jl

```diff
@@ -223,9 +223,13 @@ end
     A = randn(3,2)
     Ac = copy(A')
     b = randn(3)
+    b0 = copy(b)
     c = randn(2)
+    c0 = copy(c)
```

Comment:
You'll have to move this down below the place where `c` is overwritten.

Reply:
But then how would I check that `c` is not overwritten? `c0` has the original value of `c` which must be assigned before the call to `ldiv!`. Right?

Reply:
`c` is supposed to be overwritten in the line right below this one as it is the output array in `ldiv!`. That is the reason why you should copy `c` below that line to make sure that the calculation in lin 231 doesn't affect `c`.

Reply:
You are right. Fixed.